app/renderer/components/task_card.js

```diff
@@ -81,7 +81,12 @@ module.exports = {
         },
         run() {
             this.output = "";
-            this.task.run(this.print, () => {});
+            AppStatus.runningTasks++;
+            this.task.run(this.print, () => {   AppStatus.runningTasks--;   }
```

Comment:
Style note: the function should be multilined. 
While in this project is not yet defined and standardized (my fault) all the gaucho JS code is usually passed through a beautifier (https://www.npmjs.com/package/js-beautify) to generate a consistent style, if you are not sure of the coding style followed in this projet, feel free to try and execute that beautifier to automatically comply with the convetions

(Also, I should add a new issue to add the beautify as an npm script, to make this easier)